docs/user_guide.md

```diff
@@ -477,16 +477,68 @@ optional arguments:
   -s, --steps           Print steps.
 ~~~~~~~~~~~~~~~~~~~~~
 
-## 4. checkers mode:
+## 4. `checkers` mode
 
-List all available checkers.
+List the checkers available in the installed analyzers which can be used when
+performing an analysis.
 
-The ```+``` (or ```-```) sign before a name of a checker shows whether the checker is enabled (or disabled) by default.
+### Old `checkers` command (before `v6.0` release)
+
+**WARNING!** Usage of this command is deprecated and with the release of `v6.0`
```

Comment:
I meant to remove this warning from the user guide too before v6.0 is released. We will add it later when the new v6.0 is out and we see how we roll out the old command line or we keep compatibility.